docs/ref/contrib/contenttypes.txt

```diff
@@ -443,6 +443,32 @@ behavior; if desired, you can avoid the cascade-deletion by not using
 behavior can be provided via the :data:`~django.db.models.signals.pre_delete`
 signal.
 
+Methods on ``GenericRelation`` managers
+---------------------------------------
+
+    .. method:: remove(*objs, bulk=True)
+
+        Removes the specified model objects from the related object set and
+        deletes them from the database.
+
+    .. code-block:: python
+
+        >>> b.tags.remove(t) # Deletes TaggedItem t from database.
+
+    .. method:: clear()
+
+        Removes all items from the related object set and deletes them from the 
+        database.
+
+    .. code-block:: python
+
+        >>> b.tags.clear() # Deletes all related TaggedItems from database.
+
+In both the above examples this behavior differs from that of
+:class:`django.db.models.fields.related.RelatedManager` which has
+methods named ``remove`` and ``clear`` which disassociate the 
+related objects without deleting them from the database.
```

Comment:
IMO we don't seed a separate section I would add it to the ["Reverse generic relations"](https://docs.djangoproject.com/en/3.1/ref/contrib/contenttypes/#reverse-generic-relations), e.g.
```diff
diff --git a/docs/ref/contrib/contenttypes.txt b/docs/ref/contrib/contenttypes.txt
index 28a2e40819..45a96a2aa6 100644
--- a/docs/ref/contrib/contenttypes.txt
+++ b/docs/ref/contrib/contenttypes.txt
@@ -391,6 +391,23 @@ be used to retrieve their associated ``TaggedItems``::
     >>> b.tags.all()
     <QuerySet [<TaggedItem: django>, <TaggedItem: python>]>
 
+You can also use :meth:`~django.db.models.fields.related.RelatedManager.add`,
+:meth:`~django.db.models.fields.related.RelatedManager.create`, or
+:meth:`~django.db.models.fields.related.RelatedManager.set` to create
+relationships::
+
+    >>> ....
+
+The :meth:`~django.db.models.fields.related.RelatedManager.remove` call will
+bulk delete the specified model objects::
+
+    >>> ...
+
+The :meth:`~django.db.models.fields.related.RelatedManager.clear`
+method can be used to bulk delete all related objects for an instance::
+
+    >>> ...
+
 Defining :class:`~django.contrib.contenttypes.fields.GenericRelation` with
 ``related_query_name`` set allows querying from the related object::
 
```
In examples we can show that related objects are deleted.

Reply:
Hi @felixxm , sorry it took me so long to get back to this. I'm planning to work on it today and tomorrow. Can I clarify something? Should this section be talking about `django.db.models.fields.RelatedManager` or `django.contrib.contenttypes.fields.GenericRelation`?